desktop/libs/metadata/src/metadata/catalog/atlas_client.py

```diff
@@ -65,7 +66,25 @@ class AtlasApi(Api):
   def __init__(self, user=None):
     super(AtlasApi, self).__init__(user)
 
-    self._api_url = CATALOG.API_URL.get().strip('/') + "/api/atlas"
+    # Checking which server is active server if there are multiple else use the one thats listed
+    atlas_servers = CATALOG.API_URL.get().replace("%20 ","").replace("['","").replace("']","").replace("'","").split(',')
```

Comment:
Circle CI complains about missing whitespace after the comma, and I think the whitespace in `"%20 "` should not be there (at least it isn't there in our case, we're facing the same issue).
The full line would be
```python
atlas_servers = CATALOG.API_URL.get().replace("%20", "").replace("['", "").replace("']", "").replace("'", "").split(',')
```